ch6/error_set.zig

```diff
@@ -0,0 +1,5 @@
+const NetworkError = error{
+    ConnectionLost,
+    Timeout,
+    InvalidResponse,
+};
```

Comment:
in the old example you had 2 error sets. Are you going to use a single one right now?

Reply:
For this specific example, yes 🌞